www/com.salesforce.util.push.js

```diff
@@ -24,55 +24,53 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-// Version this js was shipped with
```

Comment:
That code lives here: https://github.com/forcedotcom/SalesforceMobileSDK-Shared/blob/master/libs/cordova.force.js#L1252
You should update the shared repo, re-run shared's [update.sh](https://github.com/forcedotcom/SalesforceMobileSDK-Shared/blob/master/tools/update.sh), then this repo's [update.sh](https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/blob/master/tools/update.sh).

Reply:
Ah - yes of course. I meant to do that :-)

Reply:
Done [here](https://github.com/forcedotcom/SalesforceMobileSDK-Shared/pull/485).